recipes/squirrel/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "3.1":
+    url: "https://github.com/albertodemichelis/squirrel/archive/v3.1.tar.gz"
+    sha256: "51942b8638a97b673e34ecf3ca50304996fa99bbdbfa7fe93d9744e6769b2f95"
+patches:
+  "3.1":
+    - patch_file: "patches/001_fix.patch"
```

Comment:
it would be nice if this had a descriptive name

Reply:
I agree!